frontend/src/components/ContextMenu.js

```diff
@@ -53,11 +53,11 @@ function ContextMenu({
         className="smart-hub--context-menu-button smart-hub--button__no-margin"
         unstyled
         aria-haspopup
-        onClick={() => updateShown((previous) => !previous)}
+        onClick={() => { updateShown((previous) => !previous); }}
         aria-label={label}
         type="button"
       >
-        <FontAwesomeIcon color="black" icon={faEllipsisH} />
+        <FontAwesomeIcon color="black" icon={faEllipsisH} title="ellipsis" />
```

Comment:
Is there a better way to address this button in the tests? I think I'd prefer using `data-testid` than a title that doesn't help the user understand what's going on.